.github/workflows/e2e-test.yml

```diff
@@ -524,14 +524,6 @@ jobs:
         run: |
           ./test/scripts/gh-actions/build-qpext-image.sh
           docker image ls
-      - name: Upload qpext image
-        uses: ishworkh/docker-image-artifact-upload@v1
-        with:
-          image: "kserve/qpext:${{ github.sha }}"
-      - name: Download qpext image
-        uses: ishworkh/docker-image-artifact-download@v1
-        with:
-          image: "kserve/qpext:${{ github.sha }}"
```

Comment:
Is this a dup?

Reply:
Since we are building the image in the same job, we don't need to upload and download the image.